samples/Playground/Tests.cs

```diff
@@ -18,7 +18,7 @@ public class TestClass
     public void Test() => TestContext.AddResultFile(@"c:\hello2");
 
     [TestMethod]
-    public void Test2() => Assert.AreEqual(1, 0, "few");
+    public void Test2() => Assert.AreEqual(0, 1, "few");
```

Comment:
Unnecessary change but it doesn't harm.

Reply:
Part of the bulk fix on other assertions, I'll leave it as-is.